python/sglang/srt/managers/tp_worker.py

```diff
@@ -245,12 +241,16 @@ def forward_step(self):
             if self.running_batch is not None:
                 # Run a few decode batches continuously for reducing overhead
                 for _ in range(global_config.num_continue_decode_steps):
-                    self.num_generated_tokens += len(self.running_batch.reqs)
+                    self.total_decoded_tokens += len(self.running_batch.reqs)
+                    # TODO (min): use cuda events for better timing measurements.
+                    start_tic = time.time()
                     self.forward_decode_batch(self.running_batch)
+                    self.total_decoding_seconds += time.time() - start_tic
```

Comment:
we want something more end-to-end, so we do not use this fine-grained measurement.

If you want more fine-grained measurements, you can implement other stats.

Reply:
sorry, I am confused. This is not much more finegrained than before. The code before just include time from idle time because calls to the print stats function. This just calls the time a bit more frequently? At least from the screenshot I have added here, the before and after measurement are pretty much the same.

Reply:
the old code also includes prefilling time.

Reply:
Understood. I guess I misunderstood the goals here. I think the old code included idle time between request and report low throughput for the first decode call. But I guess it is intended.

I prefer just close this PR since there isn't much left in the PR after this is undone. It is totally OK for me.

Reply:
@min-xu-et Thanks for your understanding.